### PR TITLE
Update: Tencent.QQ version 9.4.9.27849

### PR DESCRIPTION
**Pull request by SpecterShell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25457)

### CODE REVIEW

File: manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: Tencent.QQ
+PackageVersion: 9.4.9.27849
+PackageLocale: en-US
+Publisher: Tencent Technology (Shenzhen) Co. Ltd.
```
Comment (ImJoakim) [inline manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml:7]: image

```suggestion
Publisher: 腾讯科技(深圳)有限公司
```

File: manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PublisherUrl: https://www.tencent.com/en-us
+PublisherSupportUrl: https://support.qq.com/products/36086
+PrivacyUrl: https://ti.qq.com/agreement/qqface.html?appname=mqq_2019
+#Author: 
+PackageName: QQ
```
Comment (ImJoakim) [inline manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml:12]: image

```suggestion
PackageName: 腾讯QQ
```

File: manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+PackageIdentifier: Tencent.QQ
+PackageVersion: 9.4.9.27849
+PackageLocale: zh-CN
+Publisher: 腾讯科技(深圳)有限公司
+PublisherUrl: https://www.tencent.com/zh-cn
+PublisherSupportUrl: https://support.qq.com/products/36086
+PrivacyUrl: https://ti.qq.com/agreement/qqface.html?appname=mqq_2019
+#Author: 
+PackageName: QQ
```
Comment (ImJoakim) [inline manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.zh-CN.yaml:12]: image

```suggestion
PackageName: 腾讯QQ
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
Please see my comments above:

https://github.com/microsoft/winget-pkgs/pull/25457#discussion_r692246816
https://github.com/microsoft/winget-pkgs/pull/25457#discussion_r692247688
https://github.com/microsoft/winget-pkgs/pull/25457#discussion_r692247909

File: manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml:7]: Using Chinese in en-US manifest?

Comment (ImJoakim) [reply manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml:7]: Correct. It needs to match the ARP - [schema/1.0.0/defaultLocate#publisher](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#publisher):

> Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with winget upgrade and winget list

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**